tests/test_estimators.py

```diff
@@ -0,0 +1,8 @@
+from sklearn.ensemble import RandomForestRegressor
+
+from augurpy.estimators import create_estimator
+
+
+def test_creation():
+    """Test output of create_estimator."""
+    assert isinstance(create_estimator("random_forest_regressor"), RandomForestRegressor)
```

Comment:
Quite an easy test :)

You could also test what happens when someone enters an unexpected input aka a non-existing estimator. 
Also, you can test for values of params -> what happens when I do not pass a params value, what happens when I do?

Just simple things.

Reply:
Do you know how I can compare sklearn objects - to see if the params get assign etc? Is the best way to directly compare parameters?

Reply:
IMO asserting that the params object of the estimator is not None and asserting a specific value of the params object then to have a desired value is sufficient. No need to test every combination.